holoviews/tests/operation/testdatashader.py

```diff
@@ -19,6 +19,14 @@
 except:
     raise SkipTest('Datashader not available')
 
+try:
+    import spatialpandas
+except:
+    spatialpandas = None
+
+sp_skip = skipIf(spatialpandas is None, "SpatialPandas not available")
```

Comment:
I just about guessed that `sp_skip` meant 'spatial pandas skip' because of the context of this PR but otherwise I think it wouldn't be a bad idea to be explicit and call it `spatialpandas_skip`.

Reply:
I'll update it.